internal/livelint/check_does_image_tag_exist.go

```diff
@@ -0,0 +1,32 @@
+package livelint
+
+import (
+	"fmt"
+	"strings"
+
+	apiv1 "k8s.io/api/core/v1"
+)
+
+func (n *Livelint) checkDoesImageTagExist(container apiv1.Container) CheckResult {
+	parts := strings.Split(container.Image, ":")
+
+	image := parts[0]
+	tag := "latest"
+	if len(parts) > 1 {
+		tag = parts[1]
+	}
+
+	yes := n.askUserYesOrNo(fmt.Sprintf("Does the tag %q for the image %q exist in the docker repository?", tag, image))
```

Comment:
```suggestion
	yes := n.askUserYesOrNo(fmt.Sprintf("Does the tag %q for the image %q exist in the container repository?", tag, image))
```

Docker is just one brand. We should be generic.